src/settings/JobProfiles/ViewJobProfile.js

```diff
@@ -226,10 +208,6 @@ export class ViewJobProfile extends Component {
     this.setState({ showDeleteConfirmation: true });
   };
 
-  showRunConfirmation = () => {
```

Comment:
This method is used by the **Run** button, which is one of the action buttons (see it in `menuTemplate.js` file 71 line), do not remove it, please.

Reply:
Done!